pages/TextMessages.tsx

```
@@ -0,0 +1,1431 @@
+                              }`}
+                            >
+                              {category.name}
+                            </button>
+                          ))}
+                        </div>
+                        
+                        <div className="space-y-2 max-h-48 overflow-y-auto">
+                          {filteredTemplates.map(template => (
+                            <div 
+                              key={template.id}
+                              onClick={() => useTemplate(template.content)}
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)